### PR TITLE
fix-div-alignment for mobile view

### PR DESCRIPTION
**Pull request by Mohitpanjikar**
#82 
for normal view -
Screenshot 2024-10-05 at 11 38 03 AM
for mobile view -
Screenshot 2024-10-05 at 11 38 42 AM

### CODE REVIEW

**Review by reharri7 — APPROVED** [review]

**Comment by fernandonevarez** [comment]
Hello @Mohitpanjikar, could you also fix the section alignment so it's flush with the rest of the site? I thought setting the flex's justification to 'start' would fix the problem, but there's still a dent in the placement.

dscfsnwlcwe

star_ncdsbhjgc

File: components/infiniteLogoCarousel.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { InfiniteMovingCards } from "~/components/ui/infinite-moving-cards";
 export const InfiniteLogoCarousel = () => {
   return (
     <div className=" relative flex flex-col items-center justify-center overflow-hidden rounded-md antialiased">
-      <h2 className="w-10/12 py-4 text-center text-2xl font-bold md:text-4xl">
+      <h2 className="w-10/12 py-4 md:text-center text-2xl font-bold md:text-4xl">
```
Comment (fernandonevarez) [inline components/infiniteLogoCarousel.tsx:6]: It's best practice to also have the mobile styles in line, so add `text-left` to this className.

File: components/infiniteLogoCarousel.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { InfiniteMovingCards } from "~/components/ui/infinite-moving-cards";
 export const InfiniteLogoCarousel = () => {
   return (
     <div className=" relative flex flex-col items-center justify-center overflow-hidden rounded-md antialiased">
```
Comment (fernandonevarez) [inline components/infiniteLogoCarousel.tsx:5]: To fix the problem, update `items-center` to be `items-start md:items-center`

Comment (fernandonevarez) [inline components/infiniteLogoCarousel.tsx:5]: For formatting, remove the leading space in the className.
sdnkcnjsnjk

**Review by fernandonevarez — CHANGES_REQUESTED** [review]
Please address these issues before merging 👍

File: components/infiniteLogoCarousel.tsx (hunk 2 above)
Comment (reharri7) [reply components/infiniteLogoCarousel.tsx:5]: I don't see a diff here

Comment (reharri7) [reply components/infiniteLogoCarousel.tsx:5]: Is this resolved?

**Review by reharri7 — CHANGES_REQUESTED** [review]
It may be that I am on mobile but I am not seeing a diff where the requested changes from @fernandonevarez are resolved

**Comment by reharri7** [comment]
There are no new commits since changes were requested

File: components/infiniteLogoCarousel.tsx (hunk 2 above)
Comment (fernandonevarez) [reply components/infiniteLogoCarousel.tsx:5]: Here are some pictures for reference. 

What it is currently:
fhiewfihw

What it needs to be for mobile view:
jiuyhguyg

What it needs to be for desktop view:
chjdsc'

**Comment by fernandonevarez** [comment]
Yeah @reharri7, no changes were made. But i laid out the changes needed to be made so someone could do them pretty easily.

**Comment by reharri7** [comment]
Great! Hopefully will see some changes soon ☺️

File: components/infiniteLogoCarousel.tsx (hunk 2 above)
Comment (fernandonevarez) [reply components/infiniteLogoCarousel.tsx:5]: no not yet

**Pull request closed without merging**